lib/Net/SAML2/SP.pm

```diff
@@ -338,8 +354,16 @@ sub logout_request {
         NonEmptySimpleStr->check($nameid_format)
             ? (nameid_format => $nameid_format)
             : (),
+        ((defined $sp_name_qualifier) && NonEmptySimpleStr->check($sp_name_qualifier))
+            ? (affiliation_group_id => $sp_name_qualifier)
+            : (),
+        ((defined $name_qualifier) && NonEmptySimpleStr->check($name_qualifier))
+            ? (name_qualifier => $name_qualifier)
+            : (),
+        ((defined $include_name_qualifier))
+            ? ( include_name_qualifier => $include_name_qualifier )
+            : ( include_name_qualifier => 1),
```

Comment:
Why am I hard coding include_name_qualifier - should have a default value set if we really want this

Reply:
Let's discuss this on irc for a moment.